cpp/tests/utilities_tests/logger_tests.cpp

```diff
@@ -55,7 +55,7 @@ TEST_F(LoggerTest, DefaultLevel)
   cudf::default_logger().warn("warn");
   cudf::default_logger().error("error");
   cudf::default_logger().critical("critical");
-  ASSERT_EQ(this->sink_content(), "info\nwarn\nerror\ncritical\n");
+  ASSERT_EQ(this->sink_content(), "warn\nerror\ncritical\n");
```

Comment:
What is this?
Also, why is it an `ASSERT_EQ` vs `EXPECTS_EQ`?

Reply:
> What is this?

I'm not sure what your first question is asking. We've had these tests in cudf since the original implementation of a logger was added. This test is checking whether the expected text is being logged from the above `warn(...)` `error(...)` etc commands. The test needs to be updated because we are now hiding `info` logs by default at runtime.

> Also, why is it an `ASSERT_EQ` vs `EXPECTS_EQ`?

That is a good question. I am not sure. I would prefer not to spend time fixing anything here though since I plan to migrate all tests of logging to the `rapids-logger` repository very soon anyway. Currently every repository has its own set of logger tests so I'd rather fix issues like this in one place rather than in four.

Reply:
I was just curious what the string represents. It seemed an odd thing to rely/test on.

Reply:
Got it. The string is the actual logged output, so I think it is the right thing to test as long as we avoid issues around formatting, which we are currently doing by unsetting any formatting in the fixture.